l10n_fr_hr_ssnid_validation/views/hr_employee.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+  <record id="view_employee_form" model="ir.ui.view">
+    <field name="name">ssnid.view.employee.form</field>
+    <field name="model">hr.employee</field>
+    <field name="inherit_id" ref="hr.view_employee_form"/>
+    <field name="arch" type="xml">
+
+      <field name="user_id" position="after">
+        <field name="ssnid"/>
```

Comment:
isn't weird that field ssnid defined in core is not shown anywhere in UI in odoo?